### PR TITLE
[MNOE-474] Update review frontend

### PR DESCRIPTION
**Pull request by adamaziz15**
@alexnoox

### CODE REVIEW

File: src/app/views/marketplace/marketplace-app.html [hunk 1]
```diff
@@ -7,17 +7,22 @@
...
           <h2 class="media-heading">{{vm.app.name}}</h2>
         </div>
+        <div class="app-rating pull-left">
+          <div ng-show="vm.isRateDisplayed">
+            <average-star-rating ng-model="vm.averageRating" display-rating="true" max="5"></average-star-rating>
+          </div>
+        </div>
+        <div class="pull-left links">
+          <a ng-show="vm.anyReviews" ng-click="vm.scrollToReviews()" href="">{{vm.reviews.list.length}} {{'mno_enterprise.templates.dashboard.marketplace.show.reviews' | translate}}</a>
+          <span ng-if="vm.anyReviews && vm.anyQuestions">  |  </span>
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-app.html]: Use `&nbsp;`: `&nbsp;|&nbsp;`
I'm afraid that the code uglifier will remove those spaces after a build.

**Comment by adamaziz15** [comment]
@alexnoox replaced spaces with `&nbsp;`

**Comment by alexnoox** [comment]
@ouranos Please port locales to mno-enterprise and merge

File: src/locales/en.locale.json [hunk 2]
```diff
@@ -519,6 +521,9 @@
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_body": "Do you really want to delete your comment?",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_cancel": "Cancel",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_delete": "Delete",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment.hide": "Hide Comments",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment": "Comment",
+  "mno_enterprise.templates.dashboard.marketplace.show.comments": "Comments",
```
Comment (alexnoox) [inline src/locales/en.locale.json]: You can use pluralisation to manage this (https://angular-translate.github.io/docs/#/guide/14_pluralization)

File: src/locales/en.locale.json [hunk 3]
```diff
@@ -519,6 +521,9 @@
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_body": "Do you really want to delete your comment?",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_cancel": "Cancel",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_delete": "Delete",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment.hide": "Hide Comments",
```
Comment (alexnoox) [inline src/locales/en.locale.json]: Rename this key to `mno_enterprise.templates.dashboard.marketplace.show.hide_comments`. If a key is used (ie. `mno_enterprise.templates.dashboard.marketplace.show.comment`) we cannot reuse it by adding `.hide`. It has to be translated to YAML and this format do not accept it.

**Comment by adamaziz15** [comment]
@alexnoox updated locales and view to use pluralization

**Comment by adamaziz15** [comment]
Attempting to port locales: https://github.com/maestrano/mno-enterprise/pull/328

**Comment by adamaziz15** [comment]
@alexnoox Updated locales and created PR in mnoe. Please review.

**Comment by ouranos** [comment]
@alexnoox locales all good for me. Let's merge in `1.1` and I'll include it before my big `2.0` refactor

**Comment by alexnoox** [comment]
@ouranos Ok, let's do it

**Comment by ouranos** [comment]
Locales merged in maestrano/mno-enterprise#334

**Pull request merged**